crates/yas_core/src/contracts/yas_pool.cairo

```diff
@@ -199,6 +206,23 @@ mod YASPool {
             self.token_1.read()
         }
 
+        fn slot_0(self: @ContractState) -> Slot0 {
+            self.slot_0.read()
+        }
+
+        fn max_liquidity_per_tick(self: @ContractState) -> u128 {
+            self.max_liquidity_per_tick.read()
+        }
+
+        fn tick_spacing(self: @ContractState) -> i32 {
+            self.tick_spacing.read()
+        }
+
+        fn ticks(self: @ContractState, tick: i32) -> Tick::Info {
+            let tick_state = Tick::unsafe_new_contract_state();
+            TickImpl::get_ticks(@tick_state, tick)
+        }
```

Comment:
```suggestion
        fn tick(self: @ContractState, tick: i32) -> Info {
            let tick_state = Tick::unsafe_new_contract_state();
            TickImpl::get_ticks(@tick_state, tick)
        }
```

Reply:
Wouldn't it better to name this functions as `get_x` to provide more clarity on what they actually do?

Reply:
done, but can't remove Tick:: in return type since Info collides with another struct used in scope (Position::Info)